web/src/views/settings/SearchSettingsView.tsx

```diff
@@ -20,50 +20,53 @@ import {
   SelectItem,
   SelectTrigger,
 } from "@/components/ui/select";
+import { Trans, useTranslation } from "react-i18next";
 
-type SearchSettingsViewProps = {
+type ExploreSettingsViewProps = {
   setUnsavedChanges: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
-type SearchSettings = {
+type ExploreSettings = {
   enabled?: boolean;
   reindex?: boolean;
   model_size?: SearchModelSize;
 };
 
-export default function SearchSettingsView({
+export default function ExploreSettingsView({
   setUnsavedChanges,
-}: SearchSettingsViewProps) {
+}: ExploreSettingsViewProps) {
+  const { t } = useTranslation("views/settings");
   const { data: config, mutate: updateConfig } =
     useSWR<FrigateConfig>("config");
   const [changedValue, setChangedValue] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
 
   const { addMessage, removeMessage } = useContext(StatusBarMessagesContext)!;
 
-  const [searchSettings, setSearchSettings] = useState<SearchSettings>({
+  const [ExploreSettings, setExploreSettings] = useState<ExploreSettings>({
```

Comment:
```suggestion
  const [exploreSettings, setExploreSettings] = useState<ExploreSettings>({
```
The variable name should not be the same as the type. Use a lowercase `e` for the variable name.